app/pages/collections/list.cjsx

```diff
@@ -50,7 +50,7 @@ List = React.createClass
   listCollections: (props) ->
     query = {}
     if props.params?.collection_owner?
-      query.owner = props.params.collection_owner
+      query.current_user_roles = "owner,contributor,collaborator"
     else if props.params?.profile_name?
       query.owner = props.params.profile_name
```

Comment:
This is for the `users/:profile_name/collections` route and similarly would need a way to fetch the collections for each role that is linked the `:profile_name` user . Unless we decide that if a user is viewing the collections via the profile that they should only strictly see the collections that that the `:profile-owner` owners. To be honest, I'm not too keen on this route existing at all since it loads up the same collection list component as `/collections/:collection_owner`